docs/design/0000-00-01-architecture-of-storage-engine-pagestorage.md

```diff
@@ -0,0 +1,239 @@
+# Architecture Of Storage Engine - PageStorage
```

Comment:
change the filename to "2022-04-28-architecture-of-storage-engine-pagestorage.md"